### ISSUE TITLE
Behavior with templated messages

### ISSUE THREAD

**Issue by shabino** [abusix/rate-limiting-logback-filter#1]
Users should note that message template parameters are not considered when evaluating a log event's uniqueness.

`log.info("The value was {}", value)` will only be emitted once per interval even if the value parameters for each event differed. However, 
`log.info("The value was " + value)` would be emitted for each unique value.

I'm not calling this a bug, merely something that users should consider.

**Comment by FrederikP** [comment]
Good point. This was indeed the desired behavior since sentry's event grouping/clustering has the same characteristic (by default).
Still it's good to point it out. I'll leave this open for now.
